### ISSUE TITLE
Error: EISDIR: illegal operation on a directory, read

### ISSUE THREAD

**Issue by onetrev** [pucelle/vscode-css-navigation#69]
I've been seeing this error generated by CSS Navigation for quite some time, and I still can't figure out why. I regularly get this output:

```
[8:35:02] Client for workspace folder "mytheme" started
[8:35:03] Server for workspace folder "mytheme" started
[Error - 8:35:03 AM] Error: EISDIR: illegal operation on a directory, read
```

Two things to note. I'm using VS Code in a WSL2 environment and I use [Lando](https://lando.dev) to create my WordPress dev environments (where this error regularly appears).

**Comment by pucelle** [comment]
Really cant point out how this issue happens. I made a guess-to-fix, will release in v1.14.0, but not sure whether it would work.

**Comment by onetrev** [comment]
Amazing, thank you! I seem to not see this error as much (if at all) since I formatted and did a clean WSL2 install, so I won't be able to confirm either if it's fixed. But let's just say it is for sure fix. :)
